assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetGA4/UserCountGraph.js

```diff
@@ -93,6 +95,18 @@ export default function UserCountGraph( props ) {
 		};
 	}, [] );
 
+	const propertyID = useSelect( ( select ) =>
+		select( MODULES_ANALYTICS_4 ).getPropertyID()
+	);
+	const property = useSelect( ( select ) =>
+		propertyID
+			? select( MODULES_ANALYTICS_4 ).getProperty( propertyID )
+			: null
+	);
+	const propertyCreatedDate = property?.createTime
+		? getDateString( new Date( property.createTime ) )
+		: null;
```

Comment:
Seems like a `getPropertyCreationTime()` selector would be useful to save a bit of boilerplate, although it's arguably something we could add later...

Reply:
`getCurrentProperty()` sounds good, and adding it in a separate issue also makes sense, thanks for filing the issue!